core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -20,7 +20,6 @@ package kafka.admin
 import java.nio.charset.StandardCharsets
 import java.util.concurrent.TimeUnit
 import java.util.{Collections, Properties}
-
```

Comment:
probably nice to avoid messing with the whitespace here

Reply:
yea, we need to figure out how to stop IntelliJ from doing this 🤔

Reply:
IntelliJ has a default scalafmt config, we also have a config at checkstyle/.scalafmt.conf but it's only applied to strams-scala module.